src/CONST.ts

```diff
@@ -4615,6 +4615,25 @@ const CONST = {
                         '\n' +
                         `[Take me to workspace category settings](${workspaceCategoriesLink}).`,
                 },
+                {
+                    type: 'setupTags',
+                    autoCompleted: false,
+                    title: 'Set up tags (optional)',
+                    description: ({workspaceMoreFeaturesLink}) =>
+                        'Tags can be used if you want more details with every expense. Use tags for projects, clients, locations, departments, and more. If you need multiple levels of tags you can upgrade to a control plan.\n' +
+                        '\n' +
+                        '*Here’s how to setup tags:*\n' +
```

Comment:
Not sure if that's intentional, we have both "Set up tags" and "Setup tags"

<img width="777" alt="Screenshot 2024-09-15 at 11 00 33" src="https://github.com/user-attachments/assets/6901dd42-affe-4c33-8f2a-e24d23d51fd6">

```suggestion
                        '*Here’s how to set up tags:*\n' +
```